development/webpack/utils/plugins/ManifestPlugin/helpers.ts

```diff
@@ -29,6 +38,48 @@ export function transformManifest(args: { lockdown: boolean; test: boolean }) {
     transforms.push(removeLockdown);
   }
 
+  /**
+   * This function sets predefined flags in the manifest's _flags property
+   * that are stored in the .manifest-flags.json file.
```

Comment:
Since the the name of the file is now up to the contributor, maybe we should say something like:

```suggestion
   * that are stored in the file specified by the `MANIFEST_OVERRIDES` build variable
```

?

Reply:
Good catch ! 1e89e3375e0c58d741310b5e4042f18122ebf30c